portfolio/src/main/java/com/google/sps/translation/TranslationProvider.java

```diff
@@ -17,22 +17,63 @@
 import com.google.cloud.translate.Translate;
 import com.google.cloud.translate.TranslateOptions;
 import com.google.cloud.translate.Translation;
+import com.google.cloud.translate.Language;
+import com.google.sps.data.Comment;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
-/** Class that allows to translate text. */
+/** Class that allows to translate. */
 public class TranslationProvider {
+  public static final String DEFAULT_TARGET_LANGUAGE = "en";
+  private Translate translate;
+  private Set<String> supportedLanguageCodes;
+
+  public TranslationProvider() {
+    this.translate = 
+      TranslateOptions.newBuilder()
+        .setProjectId("gioda-step-2020")
```

Comment:
Worth to extract this as a constant as well.
Ideally, this project should be somehow retrieved from the configuration, but I'm not sure if there is an easy tool to achieve that.

Reply:
you could read it from the environment variable using `System.getEnv()` https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getenv--

Reply:
Is it stored somewhere in the env? This need to be available locally and in the cloud.